docs/framework/install/on-windows-8.md

```diff
@@ -13,22 +13,35 @@ ms.devlang: dotnet
 
 # Install the .NET Framework on Windows 8
 
-You may need the .NET Framework to run an app on Windows 8. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
+The .NET Framework is required to run many applications on Windows. You can use the following instructions to install it. You may have arrived on this page after trying to run an application and seeing the following dialog on your machine.
 
-## .NET Framework 4.7.1
+![This application could not be started](./media/this-application-could-not-be-started.png)
 
-You can install the [.NET Framework 4.7.1](https://www.microsoft.com/net/download/framework) on Windows 8.
+These instructions will help you install the .NET Framework versions you need. The .NET Framework 4.7.1 is the latest version, however, it is not supported on Windows 8.
 
-The .NET Framework 4.7.1 is included with the [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10).
+## .NET Framework 4.6.1 and later
 
-The .NET Framework 4.7.1 supports apps built for .NET Framework 4.0 or later. Other versions of the .NET Framework are available from the [Download .NET Framework](https://www.microsoft.com/net/download/framework) webpage.
+You cannot install the .NET Framework 4.6.1 or later on Windows Vista. Instead, you must upgrade to a later Windows version to use a later version of the .NET Framework.
+
+The [.NET Framework 4.7.1](https://www.microsoft.com/net/framework/versions/net471?utm_source=ms-docs&utm_medium=referral) is the latest version of the .NET Framework. It is included with the [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10) and [Windows Server 2016 Version 1709](https://docs.microsoft.com/windows-server/get-started/get-started-with-1709).
+
+## .NET Framework 4.6
+
+The [.NET Framework 4.6](https://www.microsoft.com/en-us/download/details.aspx?id=48130) is the latest support .NET Framework version on Windows 8.
```

Comment:
support --> supported
But the latest supported version is 4.6.1, according to [System Requirements](https://docs.microsoft.com/en-us/dotnet/framework/get-started/system-requirements).